tasks/create_db.yml

```diff
@@ -1,6 +1,31 @@
 ---
-# For ssl there should be a server key and server cert for the database. The key/cert files should go on all nodes. I have put them in parent dir of the
-# catalog_dir owned dbadmin_user/dbadmin_group with the key being mode 400 cert mode 444
+- name: Check if database exists 
+  command: /opt/vertica/bin/admintools -t list_db -d mon
+  sudo_user: dbadmin
+  ignore_errors: yes
+  register: create_database
 
-# In chef neither the db creation nor schema was setup in the cookbook, in Ansible I think we could create the db and start the service and let the
-# schema be defined later. I need to pull the details from some old code.
+- name: Create database 
```

Comment:
How do you intend to handle this for a cluster? Currently this will run on each node with the create step failing if the other nodes are not yet up and running.